frontstage/controllers/auth_controller.py

```diff
@@ -28,21 +28,17 @@ def sign_in(username, password):
 
     try:
         response.raise_for_status()
-        # This if statement will only be true if the 'oauth' service is the ras-rm-auth-service.  This is only
-        # meant to be a temporary measure until the switch is made and the new auth service is used.  Once
-        # the oauth2 service is switched off, we should look to tidy up the api of the auth service (As the service
-        # doesn't need a /tokens endpoint if there are no tokens) and tidy up this services interaction with it.
         if response.status_code == 204:
```

Comment:
This bit of code needs to change.  Looking specifically for a 204 was there to handle the changeover.  
After the 'successfully signed in' log line, you need to delete `return response.json()` and replace it with`return {}`
You also need to delete this if block.  so the code should look  something like 
```python
try:
    response.raise_for_status()
except requests.exceptions.HTTPError:
    <error code>

bound_logger.info('Successfully signed in')
return {}
```

Reply:
Good idea, thanks. Done.